src/simple_logger.rs

```diff
@@ -1,12 +1,13 @@
 use crate::ios::ffi_for_sanity_tests::{LOG_SENDER, CoreLogMessageThreadSafe, CoreLogLevel};
-use log::{Level, Metadata, Record};
-use log::{LevelFilter, SetLoggerError};
+// use log::{Level, Metadata, Record};
+// use log::{LevelFilter, SetLoggerError};
 use chrono::Utc;
+use log::*;
 
 static LOGGER: SimpleLogger = SimpleLogger;
 
 pub fn init() -> Result<(), SetLoggerError> {
-    log::set_logger(&LOGGER).map(|()| log::set_max_level(LevelFilter::Info))
+    log::set_logger(&LOGGER).map(|()| log::set_max_level(LevelFilter::Trace))
```

Comment:
We should add the ability to set log level from app side...

Reply:
Can you create a ticket?